packages/ui-components/src/components/single-select-dropdown/single-select-dropdown.tsx

```diff
@@ -52,6 +53,8 @@ export class KvSingleSelectDropdown implements ISingleSelectDropdown, ISingleSel
 	@Prop({ reflect: true }) minHeight?: string;
 	/** @inheritdoc */
 	@Prop({ reflect: true }) maxHeight?: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) inputSize?: EComponentSize = EComponentSize.Large;
```

Comment:
Same here

Reply:
Most of the props inside this component are used to configure the input. I believe it would be a great refactor here but merging this inside an inputConfig alongside all the other props related with the input would introduce major breaking changes both in this component and tests and in MAPS. Should we consider this refactor in another PR?

Reply:
Hum, you are correct, let's do this on another PR 😄